### ISSUE TITLE
Error regarding loss.backward() when enabling ddp

### ISSUE THREAD

**Issue by YHPeter** [teslacool/UnifiedMolPretrain#2]
We face the following problem when enabling the DDP. The environment is set up by following Dockerfile.

![image](https://user-images.githubusercontent.com/44126839/185774045-e5bae340-6f25-4257-8a67-382d24e9391f.png)

Thank you for your reply.

**Comment by OStars** [comment]
> We face the following problem when enabling the DDP. The environment is set up by following Dockerfile.
> 
> ![image](https://user-images.githubusercontent.com/44126839/185774045-e5bae340-6f25-4257-8a67-382d24e9391f.png)
> 
> Thank you for your reply.

Have you solved this problem?
